lib/nns-utils/src/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './candid';
+export * from './governance';
+export * from './ledger';
+export * from './util';
+
+import * as GovernanceDeclarations from './canisters/governance';
+import * as LedgerDeclarations from './canisters/ledger';
+
+export { GovernanceDeclarations, LedgerDeclarations };
```

Comment:
I don't like these exports, but I needed to separate them because of some types that have the same names and are conflicting

Reply:
Let's keep them in this package for now